data/tabular/blood_brain_barrier_martins_et_al/meta.yaml

```diff
@@ -0,0 +1,155 @@
+    foreign drugs. Thus the ability of a drug to penetrate the barrier to deliver
+    to the site of action forms a crucial challenge in development of drugs for the
+    central nervous system.
+targets:
+    - id: penetrate_BBB
+      description: The ability of a drug to penetrate the blood brain barrier (1) or not (0)
+      units:
+      type: boolean
+      names:
+          - noun: blood brain barrier penetration
+          - noun: ADME blood-brain barrier penetration
+          - verb: penetrates the blood brain barrier to reach the brain
+          - verb: penetrates the blood brain barrier
+          - adjective: penetrating the blood brain barrier
+          - adjective: penetrating the blood brain barrier to reach the brain
+      uris:
+benchmarks:
+    - name: TDC
+      link: https://tdcommons.ai/
+      split_column: split
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: compound_name
+      type: Other
+      names:
+          - noun: compound name
+          - noun: drug name
+          - noun: generic drug name
+      description: compound name
+license: CC BY 4.0
+links:
+    - url: https://doi.org/10.1021/ci300124c
+      description: corresponding publication
+    - url: https://rb.gy/0xx91v
+      description: corresponding publication
+    - url: https://tdcommons.ai/single_pred_tasks/adme/#bbb-blood-brain-barrier-martins-et-al
+      description: data source
+num_points: 2030
+bibtex:
+    - |-
+      @article{Martins2012,
+      doi = {10.1021/ci300124c},
+      url = {https://doi.org/10.1021/ci300124c},
+      year = {2012},
+      month = jun,
+      publisher = {American Chemical Society (ACS)},
+      volume = {52},
+      number = {6},
+      pages = {1686--1697},
+      author = {Ines Filipa Martins and Ana L. Teixeira and Luis Pinheiro
+      and Andre O. Falcao},
+      title = {A Bayesian Approach to in Silico Blood-Brain Barrier Penetration Modeling},
+      journal = {Journal of Chemical Information and Modeling}
+    - |-
+      @article{Wu2018,
+      doi = {10.1039/c7sc02664a},
+      url = {https://doi.org/10.1039/c7sc02664a},
+      year = {2018},
+      publisher = {Royal Society of Chemistry (RSC)},
+      volume = {9},
+      number = {2},
+      pages = {513--530},
+      author = {Zhenqin Wu and Bharath Ramsundar and Evan~N. Feinberg and Joseph
+      Gomes and Caleb Geniesse and Aneesh S. Pappu and Karl Leswing and Vijay Pande},
+      title = {MoleculeNet: a benchmark for molecular machine learning},
+      journal = {Chemical Science}
+templates:
+    - The molecule with the {SMILES__description} of {SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
+    - Based on the {SMILES__description} {SMILES#}, the molecule is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
+    - The {SMILES__description} {SMILES#} represents a molecule that is {penetrate_BBB#not &NULL}identified as {penetrate_BBB__names__adjective}.
```

Comment:
not sure about how the sampling code deals with spaces with this selector `{penetrate_BBB#not &NULL}`. If there is no need for a space, perhaps worth adding to the docs.

Reply:
i think I got it